src/tritonserver.cc

```diff
@@ -1215,6 +1215,20 @@ TRITONSERVER_ServerOptionsSetModelLoadThreadCount(
   return nullptr;  // Success
 }
 
+TRITONAPI_DECLSPEC TRITONSERVER_Error*
+TRITONSERVER_ServerOptionsSetLogFile(
+    TRITONSERVER_ServerOptions* options, const char* file)
+{
+#ifdef TRITON_ENABLE_LOGGING
+  std::string out_file(file);
+  LOG_SET_OUT_FILE(out_file);
```

Comment:
Should check and return error if fail to open the file

Reply:
As discussed, an error cannot be returned. Will return bool instead.

Reply:
Can you point me to the discussion?